extra/survey.xsd

```diff
@@ -0,0 +1,92 @@
+                                                        <xs:attribute type="xs:float" name="yawAtDeparture_deg" use="optional"/>
+                                                    </xs:extension>
+                                                </xs:simpleContent>
+                                            </xs:complexType>
+                                        </xs:element>
+                                        <xs:element name="scannerSettings">
+                                            <xs:complexType>
+                                                <xs:simpleContent>
+                                                    <xs:extension base="xs:string">
+                                                        <xs:attribute type="xs:string" name="template" use="optional"/>
+                                                        <xs:attribute type="xs:float" name="trajectoryTimeInterval_s" use="optional"/>
+                                                        <xs:attribute type="xs:boolean" name="active" use="optional"/>
+                                                        <xs:attribute type="xs:float" name="scanAngle_deg" use="optional"/>
+                                                        <xs:attribute type="xs:int" name="pulseFreq_hz" use="optional"/>
+                                                        <xs:attribute type="xs:float" name="scanFreq_hz" use="optional"/>
+                                                        <xs:attribute type="xs:float" name="verticalAngleMin_deg" use="optional"/>
+                                                        <xs:attribute type="xs:float" name="verticalAngleMax_deg" use="optional"/>
+                                                        <xs:attribute type="xs:float" name="headRotatePerSec_deg" use="optional"/>
+                                                        <xs:attribute type="xs:float" name="headRotateStart_deg" use="optional"/>
+                                                        <xs:attribute type="xs:float" name="headRotateStop_deg" use="optional"/>
+                                                    </xs:extension>
+                                                </xs:simpleContent>
+                                            </xs:complexType>
+                                        </xs:element>
+                                    </xs:sequence>
+                                </xs:complexType>
+                            </xs:element>
+                        </xs:sequence>
+                        <xs:attribute type="xs:string" name="name" use="required"/>
+                        <xs:attribute type="xs:string" name="platform" use="required"/>
+                        <xs:attribute type="xs:string" name="scanner" use="required"/>
+                        <xs:attribute type="xs:string" name="scene" use="required"/>
+                        <xs:attribute type="xs:int" name="seed" use="optional"/>
```

Comment:
according to the wiki, the seed can

> be an integer number, a decimal number or a string timestamp with format "YYYY-MM-DD hh:mm:ss"

or is this only valid for the command line option?

Reply:
I think it's valid for both.. so should we just set the type to `xs:string`?

Reply:
yes, would be good!